lib/navigation.js

```diff
@@ -30,23 +30,35 @@ module.exports = function () {
             let frontmatter = files[url + '/index.html']
             // if we have frontmatter for that file, create subitem
             if (frontmatter) {
-              if (frontmatter.status && frontmatter.status === 'Draft') {
```

Comment:
I think removing this should be done in another PR, or at least in a separate commit – it's not related to the intended change.

Reply:
sorry that was accidental, good catch, will revert